tools/base/libboard/board_vc/board_vc.c

```diff
@@ -118,9 +125,10 @@ fpga_result read_bmcfw_version(fpga_token token, char *bmcfw_ver, size_t len)
 
 fpga_result parse_fw_ver(char *buf, char *fw_ver, size_t len)
 {
-	uint8_t rev       = 0;
-	uint32_t var      = 0;
-	fpga_result res   = FPGA_OK;
+	uint8_t rev                = 0;
+	unsigned long int var      = 0;
```

Comment:
Why did you change to unsigned long int (64-bit) from uint32_t? You use strtoul (32-bit) and you use 32-bit format specs later.

Reply:
fixed
Changed to uint32_t